### ISSUE TITLE
Incorrect indentation of compiler-conditional accessibility modifier for module definition

### ISSUE THREAD

**Issue by roboz0r** [fsprojects/fantomas#2867]
Issue created from [fantomas-online](https://fsprojects.github.io/fantomas-tools/#/fantomas/preview?data=N4KABGBEDGD2AmBTSAuKBbBBXANogOgE74B2AxAJYBmYAIgKIBCAqgOJGlmI4DOBxJCBAokALokIkAhjg7lEJeNTlCwAQTABeMCtVC8osDQAeWsKYDU5yABpwUPqNEiA5j1RgA2vYig9UABJRAE8AB2Q0SBFRWx8hSACANxksCK841QAGO38ISABJRQVRAGUKAC9kHNyAFgywAF04gF9q33qEkPCPKLFY-wTknFSPb1ywAEY2vUgAWSljABkRREWFF1EAC37ciYAmTPqm1Va4vxmgsLTIBXgAeSplkkQSkLwd1UGUtLHcvenPvRFA8nlV6nkcFRIEcWtNzp9Lt1IgAjWCwHAfeJJb6jcFgADMAPihT4hFEADERDIAHKIADuOBWmNUokIqRhJzhHUR11R6OZgSGIzQv38NSJeRKoSk0EQjEQVFghEQAAUpIQpOhEOJCAKIKz2f5jkJTqp4Viury0RiiV9hj88QBWCVQKUyuUKpWrWB0iTQKR8QqJWD%2B5ywEh6sAGxAck1cgY8np8m3coUO8YANhdkDdsvliuVzFC4UI-sDJGDoYo4cjVBkfFjEFNQnNeUTKOtArtwvS4wA7Nncx6C4gAMI4AM8UfhngG6CiJW1%2Bsxo2ws7cy1Jzu27H23HjAAcg%2Blec9ytmiHQyIkS94K70xqb8Yum47-J3af3uQAnMf3fmvWnHAaxdOs70bMBm3aBNXygZMu13HtRT0CZsjxHMT0QNQqB1ad0HQKRI2jCCoLAVtAlgyB4I-HERTxCYpnQocAOVEpLwoOBgIjUDlxI58EUo6jU1o3tdn%2BJjMOwnU2PQDj0RAvFiNXTl1xgq4t3fYS9zo8YJkJCT3TUQhYCwRRaG4ChZJ1Ii2XvVRH0g-iLXUyJogQz8dN2cV0PmYx8ioAAVTYFBKTYlVEAB1Ch4C2SNDmUuNVJfFyoDcmjtNE-wJmdHyFn8oKFHoO9QvCqKYu2F0M3ih81zNDcUt6GJ0qQ%2Bis1yvySCoChjDuEspAXQh6GMUJlR4Hhq24vED2q%2BzapbeqkVSvpmvTXYB3agAlRA4EIeAytil0ahmoQHNI8jOgatKtJa3Sj027alXgaksCvCQHnycR0HcF0Jj4pKBIa9BcGcRlnnJJUCKcCQAvUlavyy390K2nb4FmYGKFBxBwcISHrOzaBNnVGUdQAfTpaLYr%2Buq1MWxr3JE5DVAObNfKMjVgjuQhllnfaKqm46IFOpy20oq6Ew8zK9D2RjxjmBY2akDmuYoWdntewh3s%2B776Kp%2BaaeuIGcBBlZsdx6HYeu1b-D2cTZYVpXudENGjYxk2If6vH0IJon5wkMmKe2XXoOS2mxYuCXGaEPZ9Nl3yADVvkYEQlBIFxecjaag7IhbrjDhEI7xPZvNjhZyVM%2BcJqTxRXHTw6BcaObg4B0Plst%2BGpZykvjFoWBRFYbUhpGxAxom2v%2Baz872zg7c288622tl2kGRWeVRF9BQYfCcyupICgwxINRFAvdXtfGJSapU6mQ6tTTxYZwv1rtxkXBIMuSAr8Myhf-qsGVALYEKEgMQ-UJq3gbAlJ8-1nK0yEnfDKkcIB7Duk-HU0hnCJG9KnY%2B15CDbxEHvCap9chgXARfRKV9m43xTHAm6fxEax3RqCRYmpkTwCkOOWA41U5L0xmAuyJ1G7Z31hpah4d77jHxGhWWg8JCWWKDIAA0ogRAoRAHFEKIwDU78%2BbjBIfwwWgjJ6CRnjQq2eh8Qy1yJARgE4SAAGtQQ8CMiZRQtJZyIFRujTGg9RrjRnDZQ0ZDIEUOgVQ%2Bm8C8T4ltlYxg6oWKIFoCraAhBLJUnEPAZgu9wzmWgBODU%2B8%2BETxzj0Q2xtniaJlHY7UrxgjvDhnPcxMcrHOzKXKDU0AqmlDeGCWWyTNThHgNCCBjkoEi0uq3Ux7dVD4mLlYpRKjfJqxwQ8GxUh7GOMjKheuQtRkUQarAsRESJGdysTwleZ5EAtNds8PCoQsCiBAeGHxw8-GTTPrZIpwi3yiPzuI3I%2BIF5WJkSkrUwCcBFVkjwHRxDeLDOaPYBo1Qog8HJONDweiQDNCAA)

#### Code

```fsharp
module
#if DEBUG
#else
    internal
#endif
    A = 
        let f x = x + x
```
            

#### Result

```fsharp
module
#if DEBUG
#else
internal
#endif
A =
    let f x = x + x

```
            
#### Problem description

As can be seen above, the correct indentation isn't maintained for the accessibility modifier or module name when surrounded by compiler conditionals. The below alternative is a successful workaround:

```fsharp
#if DEBUG
module A =
#else
module internal A =
#endif
```

#### Extra information

- [x] The formatted result breaks my code.
- [ ] The formatted result gives compiler warnings.
- [ ] I or my company would be willing to help fix this.

#### Options

Fantomas main branch at 1/1/1990 <- incorrect date is shown on the online editor

Default Fantomas configuration

<sub>Did you know that you can ignore files when formatting from fantomas-tool or the FAKE targets by using a [.fantomasignore file](https://fsprojects.github.io/fantomas/docs/end-users/IgnoreFiles.html)?</sub>

**Comment by dawedawe** [comment]
Hello @roboz0r,
thanks for opening this issue. We handle this correctly for [types](https://fsprojects.github.io/fantomas-tools/#/fantomas/preview?data=N4KABGBEDGD2AmBTSAuKAXAngB0QHQCc8A7AYgEsAzMAEQFEAhAVQHFCTTEAbAZ3yOIQI5YukQFiAQy7syiYvCqyhYAIIAKAJRgAvGGUqhXROjDUAHrrCWA1NcgAacFD7p0IgOY9UYANrOIUEMoABIsXB9IEXRHAKFIEIA3aQBXZDR-YIgABicsqABJBXl0AGVyAC9kPKyAFjiIAF04gF8awIbQ8PSoaNjghOSuNJ9MrIBGdsNIAFlJcwAZEUQF%2BQ90AAt%2BiYAmbM7mlTa4oOmwnB7IeXgAeUol4kRSrGNtlUHUnrHgnan3ugUdwe1U6EEgXEokAOrSmp3e5wiaEgACNYLAuG94klPqNQWAAMx-eJFPgEdAAMRE0gAcogAO5cZaYlToAhpaFHWGdBLdSKo9HM0JDEYZPG1Ilg0rYSTQRAMRCUWAERAABUkBEkAFsTOJBRBWezgochMcVHCsbykfyMUSPsMvniAKwSqBSmVyhVKlawOniaCSPhFRKwf3uWDEPVgA2IDkmrkDBGXa2Cu0ivx4gBsLsgbtl8sVyqY2FwBH9geIwdD5HDkco0j4sYgpqE5rBib5aJt3Ox9tx%2BQA7Nncx6C4gAMJcAM8MfhngG6DoJW1%2Bsxo0wk7dy1QZO2ntp76GAAcQ%2Blec9ypmiE1yN1LrrvFXhmNTfjZy3KM7Kb3DvyAE4T%2B6%2BZejOXA1neK6NmAzYdAm747t2wo-hMuR4jmp6IKolBiAQM6apqkiRtGkHQWArZdBcHYCruiF9hMkyocOQHKqUV7kHAoERuBD7Ea%2B8JwZ%2B1E4qK%2BTjL8DHoZh2EsZqbHomBeJEWunIbrBFFWgJCFCemImEuJ7qqAQsApAoNDcOQMnYYRbKPioz5QbxFpqb0ohfjRwkTOKqFzOYBSUAAKhs8ilBsSroAA6uQ8CbJG%2BxKXGKlvk5UQuYJvbucE4zOl58y%2BQF8h0A%2BwWhRFUVbC6GaxU%2B65mpuSV9Kl%2B54uMWbZT5xCUOQ5g3CWkiLgQdDmNgyo8Dw1acXih6VbZ1UtrViLOTEDVIRlg6tQASogcAEPAJXRS6tRTUIdkkWRPJ1SlmlpdpEzHutm1KvA1IpNe4h3AUYiat4LrjDxCV8UlmopFw7iMo85JKvhbjiH5FFLbRGX-qhG1bfAMxAyDyzgwQkOWdm0AbOqMrYQA%2BnSkXRb9NWqfNyWLZdjX5Hs2beQZGqYDcBBLHOu1lRNh1NDNMGJTT9X08thg7PR%2BSzPMrOSOznPkHOT0vQQb0fV9TWU7N1OXIDwPkKDiBYzj0Ow2L8MS2J0tywrXPoGjBtGybvW46h%2BOEwu4ik%2BTWza0L-0ixdCZuddPy6dL3kAGqfAwIiKMQHg85Gk3%2B6Rc2XKLIdaQeKg7J5kfzOSxkLmNccKJ4yf7fzYDHQ5bbvlnZyh7nQg7FlhfmDQsDoCwJgDUNiAjWNVd82np3tupVEW%2BlEstdLtIMss8roL68gw7gpkdcQ5BhsQqgKJequa-kilVcpVPC0mGnZ1drcQDsq024yHjEMXxCl%2BG5Sv71KTKn5sAihIFEL1May5uJxRfH9RyNN4K3wZlkHYt1n7YSkO4RI3pE5HxvAQLeIhd5jRPlke8DZIH2WgQ3JKcDm45zxDsRGkd0bkGBAsLUyJ4CSAnLAUaidF5G3AaQ8%2B8VL6B2vtPeB4sVD4hQtLAe4hzIlGkAAaUQIgbAQCShFAYBqD%2BvN8gkJskdQW6ddaUS7BIy2UipZZEgAwScxAADWwIeAGSMgoWkc5ECoyYUbAew1RqzisoaIRUCREwLEeYmhd88T4mtjYhg6omKIBoEraABBzJUjEPAJgO9wymWgJODUe8BGGIFhfHWV9Ij6wxo8bRMoHEmGeJgV4cNZ5SIjjYx2NS5QamgA0soLwQTSzSVqXA8AoRkJOhnSITd4QtxiQXGxKi1HeRVjgu4djJCOOcZGcY2Qa51woeRWBN8okIOCPiDuNi%2BHL3PIgLphtli4WwCkdAoDwx%2BKHgE8ap9rLj2mVPSJczaH5HxPPGxcj0nahAVwAqMkeB6OIRBSZzhGg1CiDwcko0fAGJAC0IAA), but as you see, not for modules.
Maybe this gets you going in the right direction to fix this yourself. Most likely we would need to apply the same trick here as for types.

**Comment by dawedawe** [comment]
To help a bit more, you'd need to do the same as in the fix for #628 but [here](https://github.com/fsprojects/fantomas/blob/a53120307994ed0e7fcea66be44fec296188ff22/src/Fantomas.Core/CodePrinter.fs#LL3714C38-L3714C38)
